dataflowAPI/rose/semantics/SymEvalSemantics.h

```diff
@@ -42,10 +42,6 @@ namespace rose {
                         return SValuePtr(new SValue(r, addr));
                     }
 
-                    static SValuePtr instance(size_t nbits, uint64_t num) {
```

Comment:
What should the instance() method creating an SValuePtr be? Is it redundant with something else on 64-bit?